Item.h

```diff
@@ -3,15 +3,26 @@
 #include <string>
 
 struct Character;
+
 struct Item
 {
-    virtual ~Item() { }
+    //Destructor
+    virtual ~Item();
+
+    //pure virtual function
     virtual void use(Character*) = 0;
-    Item(const std::string& name_, int effect ) : name(name_), boost(effect) { }
-    
-    inline const std::string& getName() { return name; }
-    inline int getBoost() const { return boost; }
+
+    //Constructor
+  //  MOve implementations to cpp file. 
+  
+    //YOU STILL NEED THE DECLARATIONS
+
```

Comment:
Do you really need to keep any of these comments?   This stuff should be obvious by now..
- [ ] clean up unnecessary comments.